bin/install

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env sh
+set -e
+
+# This script is meant for quick & easy install via:
+#   $ sudo sh -c "$(curl -fsSL https://raw.githubusercontent.com/jderusse/docker-dns-gen/master/bin/install)"
+
+
+C1='\033[0;31m'
+C2='\033[0;33m'
+C3='\033[0;32m'
+C4='\033[0;36m'
+C5='\033[0;35m'
+C0='\033[0m'
+
+if [ "$(id -u)" -ne 0 ]; then
+  echo "${C1}This script must be run by root${C0}"
+  echo "${C2} running ${C3}sudo $0 $@${C0}"
+  sudo $0 "$@"
+  exit 0
+fi
+
+GATEWAY=$(/sbin/ifconfig docker0 | grep "inet" | head -n1 | awk '{ print $2}' | cut -d: -f2 || echo "")
```

Comment:
Or `ip -4 addr show docker0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}'`

Reply:
thank you. Fixed